### ISSUE TITLE
Layout/ClosingHeredocIndentation error occurred

### ISSUE THREAD

**Issue by krororo** [rubocop-hq/rubocop#5965]
`Layout/ClosingHeredocIndentation` error occurred with empty heredoc.

--------

## Expected behavior

No error.

## Actual behavior

```
An error occurred while Layout/ClosingHeredocIndentation cop was inspecting /tmp/hoge.rb:1:5.
undefined method `>=' for nil:NilClass
..../lib/rubocop/cop/layout/closing_heredoc_indentation.rb:58:in `on_heredoc'
..../lib/rubocop/cop/mixin/heredoc.rb:12:in `on_str'
..../lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
..../lib/rubocop/cop/commissioner.rb:105:in `with_cop_error_handling'
..../lib/rubocop/cop/commissioner.rb:56:in `block in trigger_responding_cops'
..../lib/rubocop/cop/commissioner.rb:55:in `each'
..../lib/rubocop/cop/commissioner.rb:55:in `trigger_responding_cops'
..../lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
..../lib/rubocop/ast/traversal.rb:101:in `block in on_send'
..../lib/rubocop/ast/traversal.rb:99:in `each'
..../lib/rubocop/ast/traversal.rb:99:in `each_with_index'
..../lib/rubocop/ast/traversal.rb:99:in `on_send'
..../lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
..../lib/rubocop/ast/traversal.rb:12:in `walk'
..../lib/rubocop/cop/commissioner.rb:45:in `investigate'
..../lib/rubocop/cop/team.rb:114:in `investigate'
..../lib/rubocop/cop/team.rb:102:in `offenses'
..../lib/rubocop/cop/team.rb:44:in `inspect_file'
```

## Steps to reproduce the problem

Run RuboCop on the following:

```ruby
puts <<~FOO

FOO
```

## RuboCop version

```
$ rubocop -V
0.57.1 (using Parser 2.5.1.0, running on ruby 2.4.3 x86_64-linux)
```
